File: Prafful-Bisht/DAY-2/Day-2-A.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        if(nums.size()==1){
            return nums[0];
        }
        int ans =0;
        for(int i=0;i<nums.size();i++){
            ans = ans^nums[i];
        }
        return ans;

    }
};

File: Mukul Bhalla/Day 2/Day 2 A.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        counter = []
        for i in nums:
            counter.append(nums.count(i))

        for k in counter:
            if (k != 2):
                return nums[counter.index(k)]
